docs/data-sources/kafka_cluster_v2_instance.md

```diff
@@ -39,7 +39,7 @@ The following terms are used to describe attributes in the schema of this data s
 <br>Status of the cluster.<br><br>
 *___kafka_version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka to run on the cluster. Available versions: <ul> <li>`2.7.1`</li> <li>`2.8.1`</li> <li>`3.0.0`</li> <li>`3.1.1`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka to run on the cluster. Available versions: <ul> <li>`3.0.2`</li> <li>`3.1.2`</li> <li>`2.8.2`</li> <li>`2.7.1`</li> </ul><br><br>
```

Comment:
Why the orders look so weird in the new file

Reply:
Thats the autogenerated doc by the tool. Its in a descending order.